broskichat-app/src/App.test.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import Header from './Components/Partials/Header';
+import Footer from './Components/Partials/Footer';
```

Comment:
@mattkim8 no need to import it unless we're testing it. Also dont worry about testing in general until you're more comfortable or start working on more complex javascript-y tickets 💪